Automattic/jetpack
# My Jetpack Protect Card: Move tooltips to the headings.

To maintain consistency with the VideoPress card, This PR moves the position of the info-icon tooltips up next to the column headings, instead of down where the data is.  See screenshots:

VideoPress Card: Note the tooltips are next to the headings. This is how we want the Protect card to be.

![Image](https://github.com/user-attachments/assets/88d12c2c-4245-4f76-aa77-87e5ac242324)


Protect Card (Before this PR): Note that the tooltips are down within the data. In this PR we move the tooltips up to the headings, just like the VideoPress card above.

![Image](https://github.com/user-attachments/assets/2af837bc-e803-4072-96b8-fce5535dc5be)




Fixes https://github.com/Automattic/jetpack-marketing/issues/945

### Other information:


## Jetpack product discussion



The info-icon tooltips in the My Jetpack Protect card were initially added as part of: --> PT: Add value to the Protect product card in My Jetpack: pbNhbs-aP6-p2

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Checkout this branch on the Protect plugin via the Jetpack Beta plugin or on your local dev environment.
* Make sure Jetpack is connected (at least "site" connected).
* Open the Jetpack --> Protect page.
* Click/open the "Firewall" tab.
* Toggle "Off" Brute Force Protection.
* Open the My Jetpack page.
* View the Protect card. All 3 columns (Scan, Auto-firewall, & Logins blocked) should be showing an info-icon tooltip.
* Verify that the tooltips are positioned up in the headings, just to the right of each heading. (See Screenshots below:)
* Verify the tooltips look correct and are working properly.

**BEFORE:**

![Screen Shot 2024-08-21 at 14 16 38](https://github.com/user-attachments/assets/8f7a6fa4-33d4-457b-bcce-d27d4d709040)

.

**AFTER:**

![Screen Shot 2024-08-21 at 14 12 02](https://github.com/user-attachments/assets/d5502f49-8022-48e0-b4ff-c27051656b7c)
